src/js/plugins/menu_toggle/plugin.menu_toggle.js

```diff
@@ -7,84 +8,92 @@
       if (br.refs.$brTwoPageView) {
         br.refs.$brTwoPageView[0].removeEventListener('click', toggleMenuIfCenterClick, true);
       }
     }
+
+    var toggleNav = function toggleNav (br) {
+      var menuIsShowing = br.navigationIsVisible();
+      if (menuIsShowing) {
+        br.hideNavigation();
+      } else {
+        br.showNavigation();
+      }
+    }
   
     var toggleMenuIfCenterClick = function toggleMenuIfCenterClick(br, e) {
-      var menuManager = br.fullscreenMenu;
-      var bookWidth = e.currentTarget.offsetWidth;
-      var leftOffset = e.currentTarget.offsetLeft
-      var bookEndPageFlipArea = Math.round(bookWidth / 5);
+      var book = e.currentTarget;
+      if (!book) {
+        return;
+      }
+      var bookWidth = book.offsetWidth;
+      var leftOffset = book.offsetLeft
+      var bookEndPageFlipArea = Math.round(bookWidth / 3);
       var leftThreshold = Math.round(bookEndPageFlipArea + leftOffset); // without it, the click area is small
       var rightThreshold = Math.round(bookWidth - bookEndPageFlipArea + leftOffset);
       var isCenterClick = (e.clientX > leftThreshold) && (e.clientX < rightThreshold);
       
       if (isCenterClick) {
-        if (menuManager.menuIsShowing) {
-          br.hideNavigation();
-          menuManager.menuIsShowing = false;
-        } else {
-          br.showNavigation();
-          menuManager.menuIsShowing = true;
-        }
+        toggleNav(br);
         event.stopPropagation();
       }
     }
-  
+
     function registerEventHandlers(br) {
+
       var brContainer = document.querySelector('.BRcontainer') || {};
+      var mainBRWrapper = document.querySelector('.BookReader') || {};
       var firstChild = brContainer.firstChild;
   
       if (firstChild) {
-        firstChild.addEventListener('click', toggleMenuIfCenterClick.bind(this, br), true);
+        firstChild.addEventListener('click', function (e) { 
+          toggleMenuIfCenterClick(br, e);
+        }, true);
       }
-    }
 
-    BookReader.prototype.initFullScreenToggle = function brInitFullScreenToggle(e) {
-      $(document).on('BookReader:fullscreenToggled', function (e) {
-        if (this.isFullscreen()) {
-          this.fullscreenMenu.onEnterFullscreen(this);
-        } else {
-          this.fullscreenMenu.onExitFullscreen(this);
+      var toggleAtBackgroundClick = function toggleAtBackgroundClick (e) {
+        const isBackground = $(event.target).hasClass('BookReader')
+          || $(event.target).hasClass('BRcontainer')
+          || $(event.target).hasClass('BRemptypage')
+          || $(event.target).hasClass('BRpageview');
+        if (isBackground) {
+          toggleNav(br);
         }
-      }.bind(this));
-  
-      var fullscreenEventRegister = function fullscreenEventRegister (e) {
-        if (this.isFullscreen()) {
-          registerEventHandlers(this);
-        }
-      }.bind(this);
+      };
+      mainBRWrapper.addEventListener('click', toggleAtBackgroundClick, true)
```

Comment:
suggestion: Can we collapse this into:

```js
$(mainBRWrapper).on(
  'click',
  '.BookReader, .BRcontainer, .BRemptypage, .BRpageview',
  () => toggleNav(br)
);
```

DRYs it up and makes it easier to read.